website/assets/scss/_docs.scss

```diff
@@ -0,0 +1,54 @@
+.docs {
+    ul {
+        list-style: disc;
+        margin-left: 1rem;
+    }
+    p + p {
+        margin-top: var(--of--spacer--md);
+    }
+    .td-sidebar-nav .td-sidebar-link__page {
+        font-weight: normal;
+    }
+    h1,h2,h3,h4,h5,h6 {
+        color: var(--of--Color--brand--200);
+        font-weight: 800;
+        letter-spacing: 2px;
+    }
+    
+    h5, h6 {
+        font-size: var(--of--FontSize--xs);
+    }
+    h4 {
+        font-size: var(--of--FontSize--sm);
+    }
+    h3 {
+        font-size: var(--of--FontSize--md);
+    }
+    h2 {
+        font-size: var(--of--FontSize--md);
+    }
+    h1 {
+        font-size: var(--of--FontSize--lg);
```

Comment:
@matthewcarleton, 

I just checked that it broke the left menu. See that it has now a wrong organization. 

After this PR: 

<img width="251" alt="Screenshot 2020-04-24 at 12 54 25" src="https://user-images.githubusercontent.com/7708031/80209910-c4662b80-862a-11ea-987e-e5d79c99332c.png">

Before and how it should be: 

<img width="292" alt="Screenshot 2020-04-24 at 12 55 44" src="https://user-images.githubusercontent.com/7708031/80210018-ee1f5280-862a-11ea-8db8-ad51c6381c74.png">

Would be possible to fix it? 

c/c @asmacdo @jmrodri

Reply:
Ah yup, I can fix it in my current PR, just didn’t scope the styles to just the content area.